manager/integration/tests/test_migration.py

```diff
@@ -220,7 +223,70 @@ def test_migration_with_rebuilding_replica(clients, volume_name):  # NOQA
     7. Observe volume migrated to node 2 (single active engine).
     8. Validate initially written test data.
     """
-    return
+    # Step 1
+    client = get_random_client(clients)
+    current_host = common.get_self_host_id()
+    host1, host2 = get_hosts_for_migration_test(clients) # NOQA
+
+    # Step 2
+    volume = client.create_volume(name=volume_name, size=str(2 * Gi),
+                                  numberOfReplicas=3,
+                                  backingImage="",
+                                  accessMode="rwx", migratable=True)
+    volume = common.wait_for_volume_detached(client, volume_name)
+    volume.attach(hostId=current_host)
+    volume = common.wait_for_volume_healthy(client, volume_name)
+
+    volume_endpoint = get_volume_endpoint(volume)
+    write_volume_dev_random_mb_data(volume_endpoint,
+                                    1, 1500)
```

Comment:
Is 1M data enough for replica to rebuild while the volume got attached to another node and we can assert 'there should be no replica created before rebuilding complete' ?

Reply:
Hello @khushboo-rancher , 
I use 1500M to make sure the volume attached on origin node is in rebuilding status and then we can do migration to target node. If use small data size, rebuild on origin node will finish soon and hard to match the scenario test case wanted.